### PR TITLE
Add a command line option to show the version

### PR DESCRIPTION
**Pull request by rishiloyola**
Reference bug - #54 
This pull request contains new command line argument to print the version of buildifier.
**Example**
```
$./buildifier --version
buildifier version: 0.4.3 
buildifier scm revision: dd416ef7e19824ad568468fb7cf9f2e9d49c3e8f
```

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

**Comment by rishiloyola** [comment]
I signed it!

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

File: buildifier/buildifier.go [hunk 1]
```diff
@@ -83,6 +86,10 @@ func main() {
 	flag.Parse()
 	args := flag.Args()
 
+	if *version {
+		fmt.Printf("buildifier version: %s \n", BUILDIFIER_VERSION)
+		fmt.Printf("buildifier scm revision: %s", BUILD_SCM_REVISION)
```
Comment (pmbethe09) [inline buildifier/buildifier.go]: After this I think you should exit if len(args) == 0

**Comment by rishiloyola** [comment]
@pmbethe09 Have you passed `--workspace_status_command=./status.sh` argument while building it?
**Example**
```
bazel build --workspace_status_command=./status.sh //buildifier
```

File: buildifier/buildifier.go (hunk 1 above)
Comment (rishiloyola) [inline buildifier/buildifier.go]: Yes, that makes sense.

**Comment by rishiloyola** [comment]
@jayconrod can you review this?

**Comment by pmbethe09** [comment]
LGTM, @jayconrod can you double check this one?

File: status.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+buildifier_tags=$(git describe --tags)
+IFS='-' read -a parse_tags <<< "$buildifier_tags"
+if [[ $? != 0 ]];
```
Comment (jayconrod) [inline status.sh]: Nit: would "set -e" at the top work instead of these if-statements?

File: status.sh [hunk 3]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```
Comment (jayconrod) [inline status.sh]: This file should be executable. The build failed with status 126 without it.

Comment (rishiloyola) [inline status.sh]: What is the solution for this issue?
Is there any why we can give access to it using BUILD file?

Comment (jayconrod) [inline status.sh]: Git keeps track of whether a file is executable. You should be able to chmod +x it, then add and commit it.

Comment (rishiloyola) [inline status.sh]: Done. I pushed my code.

File: status.sh (hunk 2 above)
Comment (rishiloyola) [inline status.sh]: Are you talking about argument of read command?

Comment (jayconrod) [inline status.sh]: `set -e` should cause the script to exit with a failure status when any later command exits with a failure status. That frees you from having to check `$?`.

Comment (rishiloyola) [inline status.sh]: Done.

**Comment by damienmg** [comment]
This PR is causing the build to fails: http://ci.bazel.io/job/buildifier/469/.

@pmbethe09 I remind you can ask the CI to vet PR by saying "test this please"

**Pull request merged**